### DISCUSSION TITLE
segmentation fault when running npm run dev

### DISCUSSION THREAD

**Discussion by Joshxing** [nodejs/help#4995] [Q&A]
### Version

16.14.2

### Subsystem

npm

### What steps will reproduce the bug?

![image](https://user-images.githubusercontent.com/53200365/197230264-108ee414-cacd-4a89-83c1-feb8f44573d7.png)
when i run npm run dev this error show i'm using next js framework for this

### What do you see instead?

![image](https://user-images.githubusercontent.com/53200365/197230488-c971074a-5a3f-4e54-82ac-af97263d0a76.png)

### Additional information

how to fix this segmentation fault in npm

**Comment by MrJithil** [comment]
Need more information about the issue.

**Comment by bnoordhuis** [comment]
I've converted this to a discussion. Can change it back to a bug report if it turns out it's a bug in node itself and not in, for example, a module that gets loaded.
